cli/src/main.rs

```diff
@@ -103,8 +99,7 @@ enum SubCommand {
 	},
 
 	/// Run CI-specific commands
-	#[clap(alias = "ci")]
-	ContinuousIntegration {
+	CI {
```

Comment:
Is there a big reason to change this one out? It seems it's only used in one other place, and although this acronym is widely known, I think there's a chance that the full name could still help disambiguate in certain cases.